canvas_modules/common-canvas/package.json

```diff
@@ -59,7 +59,7 @@
     "@babel/plugin-transform-runtime": "7.11.5",
     "@babel/preset-env": "7.11.5",
     "@babel/preset-react": "7.10.4",
-    "@carbon/react": "^1.52.0",
+    "@carbon/react": "1.62.0-rc.0",
```

Comment:
This contains the fix for https://github.com/carbon-design-system/carbon/issues/16948